### PR TITLE
Add models and schemas for participants and events

### PR DESCRIPTION
**Pull request by F-Ruxton**

### CODE REVIEW

File: app/api/api_v1/endpoints/countries.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-from typing import Any, List
+from typing import Any
 
-from fastapi import APIRouter, Depends, HTTPException
-from sqlalchemy.orm import Session
+from fastapi import APIRouter, Depends
+from requests import Session
```
Comment (F-Ruxton) [inline app/api/api_v1/endpoints/countries.py]:
```suggestion
from sqlalchemy.orm import Session
```

File: app/api/api_v1/endpoints/events.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+from typing import Any
+
+from fastapi import APIRouter, Depends
+from requests import Session
```
Comment (F-Ruxton) [inline app/api/api_v1/endpoints/events.py]:
```suggestion
from sqlalchemy.orm import Session
```

File: app/api/api_v1/endpoints/participants.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
+from typing import Any
+
+from fastapi import APIRouter, Depends
+from requests import Session
```
Comment (F-Ruxton) [inline app/api/api_v1/endpoints/participants.py]:
```suggestion
from sqlalchemy.orm import Session
```

**Review by NicolasZanotti — COMMENTED** [review]
I am getting this error:

`ImportError: cannot import name 'EventModelWebsocket' from 'app.schemas.event'`

Do I need to generate these schemas?

File: app/schemas/event.py [hunk 4]
```diff
@@ -1,91 +1,43 @@
...
-    active: bool = True
-    uid: Optional[UID] = None
 
+# Shared properties
+class EventBase(BaseModel):
+    name: Optional[str] = None
+    lon: Optional[float] = None
+    lat: Optional[float] = None
 
-class EventModelWebsocket:
```
Comment (F-Ruxton) [inline app/schemas/event.py:24]: @NicolasZanotti the problem is that I've removed EventModelWebsocket but not pushed the changes that replace it. Sorry about that! Do you have time for a very short call about a design decision I was mulling over here?

Comment (F-Ruxton) [reply app/schemas/event.py:24]: I've added the class back in for the time being, which should mean this PR is ready to land. We can talk more about the websocket/request flow going forwards in the meeting and make changes separately.

**Pull request merged**